pkg/utils/render/json.go

```diff
@@ -0,0 +1,30 @@
+package render
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+// JsonRender struct is a general wrapper response body
+type JsonRender struct {
+	Data interface{} `json:"data"`
+}
+
+// Render implements from general interface Render
+func (r JsonRender) Render(w http.ResponseWriter) (err error) {
```

Comment:
A recommendation just to have in mind, IMHO it's better to have an explicit error and return it that's a naked return.

In this case, is a small method and it's easy to identify what you want to return, but in medium and large size functions it might be a bad idea to have naked returns.

```suggestion
func (r JsonRender) Render(w http.ResponseWriter) error {
```

and in line #18, `return err`